.gitignore

```diff
@@ -33,3 +33,4 @@ yarn-error.log*
 # typescript
 *.tsbuildinfo
 next-env.d.ts
+.env
```

Comment:
Next.js uses the `.env.local` file by default to store environment variables. You can save your variables there and then remove this file: [Next.js Environment Variables Documentation](https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables).

Reply:
Done...

Reply:
Amazing